include/sound/sof/info.h

```diff
@@ -33,6 +33,9 @@ enum sof_ipc_ext_data {
 	SOF_IPC_EXT_CC_INFO		= 2,
 	SOF_IPC_EXT_PROBE_INFO		= 3,
 	SOF_IPC_EXT_USER_ABI_INFO	= 4,
+	SOF_IPC_EXT_CONFIG_DATA		= 5,
+	SOF_IPC_EXT_PLATFORM_CONFIG_DATA 	= 6,
+
 };
```

Comment:
It can't be here - those elements are only defined for ext_manifest, not for sof_ipc_ext_data

Reply:
got it